### DISCUSSION TITLE
script for hard driver consolidation?

### DISCUSSION THREAD

**Discussion by AlexandreCA** [tteck/Proxmox#1733] [Help Needed]
Hello
This is a question I ask myself because I think it's a real hell to implement when you're new to proxmox but is there a script that does this kind of thing?

I have in my possession 4 hard drives of 1TB plus 1 TB but in flash memory (nvme). My idea was to group the 4 hard drives together to make Proxmox only support 4TB. The 1TB in NVME, I wanted to use it as cache. For example: Plex.

The purpose of all this is that I want to mount 4 TB, on different containers that need a common disk.

In short, I try to see what is possible or not to do. Thank you
